packages/tldraw/src/lib/ui/components/Minimap/DefaultMinimap.tsx

```diff
@@ -15,17 +16,18 @@ import { MinimapManager } from './MinimapManager'
 /** @public */
 export function DefaultMinimap() {
 	const editor = useEditor()
+	const container = useContainer()
```

Comment:
for my own knowledge: is it slightly easier/better to just do `editor.getContainer()` or does `useContainer()` provide some extra reactivity that i'm not aware of?

Reply:
Looks like it's basically the same thing? We avoid one anonymous function call by using the hook.
![CleanShot 2024-04-26 at 10 47 03](https://github.com/tldraw/tldraw/assets/2523721/620f8ff7-c117-4c4a-94c8-e2c2eba2d5f7)

Reply:
gotcha - i guess it feels like `useContainer` is ever-so-slightly more heavyweight than just calling `getContainer` but, meh